test/templates/message-broker-with-entities/.jhipster/Foo.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
old file, as some properties are missing. try to generate a new entity, it's better